source/ios/tools/AdaptiveCards.podspec

```diff
@@ -27,21 +32,23 @@ Pod::Spec.new do |spec|
   end
 
   spec.subspec 'AdaptiveCardsPrivate' do | sspec |
-    sspec.source_files = 'source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/PrivateHeaders/*.h'
+    sspec.source_files = 'source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/PrivateHeaders/**/*.{h,m,mm}'
     sspec.header_mappings_dir = 'source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/PrivateHeaders/'
     sspec.private_header_files = 'source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/PrivateHeaders/*.h'
   end
 
+  spec.subspec 'UIProviders' do | sspec |
```

Comment:
should the name be FlucentUIAppExtension other than UIProviders?

Reply:
we will have more third-party libraries.